prov/opx/include/rdma/opx/fi_opx_hfi1_packet.h

```diff
@@ -43,7 +43,10 @@
 #include "rdma/fabric.h"	/* only for 'fi_addr_t' ... which is a typedef to uint64_t */
 #include "rdma/opx/fi_opx_addr.h"
 
-
+#ifndef PAGE_SIZE
+/* 4K pages default */
+#define PAGE_SIZE 4000
+#endif
```

Comment:
Should this be 4096 instead of 4000? This may break some things in OPX that assume PAGE_SIZE is a power of 2, such as alignment checking/adjustment.

Please also surround this with an `#if defined(__riscv)`

Reply:
Ah right, thanks. Will make the proper correction.